src/main/java/goodgenerator/blocks/tileEntity/YottaFluidTank.java

```diff
@@ -396,6 +410,18 @@ public void construct(ItemStack stackSize, boolean hintsOnly) {
         }
     }
 
+    @Override
+    public boolean onSolderingToolRightClick(
+            byte aSide, byte aWrenchingSide, EntityPlayer aPlayer, float aX, float aY, float aZ) {
+        if (aSide == getBaseMetaTileEntity().getFrontFacing()) {
+            voidExcessEnabled ^= true;
+            aPlayer.addChatMessage(new ChatComponentTranslation(
+                    voidExcessEnabled ? "interaction.voidexcess.enabled" : "interaction.voidexcess.disabled"));
```

Comment:
where are these translation keys located at?

Reply:
They existed in gt++ so I reused them since we depend on it. Would you rather have own translations in goodgen?

Reply:
gtpp is the soft dep of goodgen. so you would better add these keys here as well.